### DISCUSSION TITLE
Scripts section reorganization

### DISCUSSION THREAD

**Discussion by euu2021** [freeplane/docs#147] [General]
It took me some time to understand how mnbooks works, but I got it, after reading their documentation. The lesson is: reading the documentation makes life easier 🤣🤣

I organized the Scripts section, in a way that guides the user through the scripts related pages. I think that, now, it has a scalable structure.

I don't like the way mdbooks lays the sidebar outline, because the user can easily miss the first page of the section by expanding the outline and going straight to the child page, like this:

![chrome_bLPqxnBDrS](https://github.com/freeplane/docs/assets/77707706/b0c54598-dceb-444a-9667-71d54b4fd549)

So, I included a "Start Here" in the page title, which is not ideal. There may be better ways of doing it.

**Comment by euu2021** [comment]
For some reason, this new page is giving Document not found (404): https://docs.freeplane.org/scripting/Setting_IDE_for_debugging.html 

I can't see what is wrong with it 🤔

**Reply by macmarrum** [reply]
A link to the page was missing in SUMMARY.md.

I've taken the liberty to correct it -> https://github.com/freeplane/docs/commit/48ccb144fe33ec708e07cec55c77d5c56ca94ef1

**Reply by euu2021** [reply]
Oh, thanks. I forgot it.
